pkg/logging/log_test.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2024 API Testing Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logging
+
+import (
+	"errors"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func TestZapLogLevel(t *testing.T) {
+	level, err := zapcore.ParseLevel("warn")
+	if err != nil {
+		t.Errorf("ParseLevel error %v", err)
+	}
+	zc := zap.NewDevelopmentConfig()
+	core := zapcore.NewCore(zapcore.NewConsoleEncoder(zc.EncoderConfig), zapcore.AddSync(os.Stdout), zap.NewAtomicLevelAt(level))
+	zapLogger := zap.New(core, zap.AddCaller())
+	log := zapLogger.Sugar()
+	log.Info("ok", "k1", "v1")
+	log.Error(errors.New("new error"), "error")
+}
+
+func TestLogger(t *testing.T) {
+
```

Comment:
Prefer to have a consistent coding style. Do you have a strong reason to keep this empty line?

Reply:
Just my personal habits, I'll update it in the next commit.